src/LivewireServiceProvider.php

```diff
@@ -84,7 +90,7 @@ public function registerRoutes()
 
     public function registerViews()
     {
-        $this->loadViewsFrom(__DIR__.DIRECTORY_SEPARATOR.'views', 'livewire');
+        $this->loadViewsFrom(__DIR__.DIRECTORY_SEPARATOR.'views', config('livewire.view-path'));
```

Comment:
second param default here? Not sure, just asking.

Reply:
about to commit it, yes.